pkg/worker/config/config.go

```diff
@@ -1,4 +1,4 @@
-package worker
+package workerConfig
```

Comment:
I am on the side of keeping the worker's config in the same package. If worker config going to need some other shared dependecy I would suggest to move them to a third party package ie. `pkg/monitoring`

Reply:
Yes, I think all config are kept in the same config place.